content/applications/hr/fleet/configuration.rst

```diff
@@ -57,109 +57,127 @@ for existing available cars in this field.
 
 .. _fleet/configuration/manufacturers:
 
+Models
+======
+
 Manufacturers
-=============
+-------------
 
-Odoo *Fleet* comes pre-configured with sixty-six commonly used car and bicycle manufacturers in the
-database, along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet
-app --> Configuration --> Manufacturers`.
+Odoo *Fleet* comes pre-configured with sixty-six commonly used car manufacturers in the database,
+along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet app -->
+Configuration --> Models: Manufacturers`.
 
 The manufacturers appear in a list view in alphabetical order. Each manufacturer's card lists how
-many specific models are configured for each particular manufacturer. Odoo comes with forty six
-pre-configured :ref:`models <fleet/configuration/models>` from four major auto manufacturers and one
-major bicycle manufacturer: Audi, BMW, Mercedes, Opel (cars), and Eddy Merckx (bicycle).
+many specific models are configured for each particular manufacturer. Odoo comes with forty five
+pre-configured :ref:`models <fleet/configuration/models>` from four major auto manufacturers: Audi,
+BMW, Mercedes, and Opel.
 
 .. image:: configuration/manufacturer.png
    :align: center
    :alt: Manufacturer card with the amount of models listed.
 
+.. _fleet/configuration/add-manufacturer:
+
 Add a manufacturer
-------------------
+~~~~~~~~~~~~~~~~~~
+
+To add a new manufacturer to the database, click :guilabel:`New`, and a blank manufacturer form
+loads.
+
+First, type the name of the manufacturer in the :guilabel:`Name` field. This is the only required
+field.
+
+Next, if a photo is available, hover over the image box and click the :guilabel:`✏️ (Pencil)` icon
+that appears in the lower-left corner.
 
-To add a new manufacturer to the database, click :guilabel:`Create`. A manufacturer form will load.
-Only two pieces of information are needed, the :guilabel:`Name` of the manufacturer, and the logo.
-Type the name of the manufacturer in the name field, and select an image to upload for the logo.
-When the information is entered, click :guilabel:`Save`.
+A file explorer window pops up. Navigate to the file, then click :guilabel:`Open` to select it.
+
+The manufacturer form automatically saves as data is entered. However, the form can be saved
+manually at any time by clicking the :guilabel:`Save manually` option, represented by a
+:guilabel:`(cloud with an upwards arrow)` icon.
 
 .. _fleet/configuration/models:
 
-Vehicle Models
-==============
+Models
```

Comment:
This contains the same name as the h2 above, was that on purpose?

Reply:
Yes, because the config menu changed, so now there is a "Models" section in the drop-down, then nested under that is Manufacturers, Models, and Categories. I wanted all the nested items to appear how they do in the DB in the right-side menu, I kept this. I left "Vehicle Models' as the heading because I didn't want there to be Models and also the same Models, so I added/kept Vehicles in the first main section, so it was understood the items were beneath that masthead.